build.sbt

```diff
@@ -1,16 +1,93 @@
-import uk.gov.hmrc.DefaultBuildSettings.integrationTestSettings
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import play.core.PlayVersion
+import sbt.Tests.{Group, SubProcess}
+import uk.gov.hmrc.DefaultBuildSettings.{addTestReportOption, defaultSettings, integrationTestSettings}
 import uk.gov.hmrc.SbtArtifactory
 import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
 
 val appName = "digital-comms-dynamic-stub"
 
+lazy val appDependencies: Seq[ModuleID] = compile ++ test()
+
+val compile = Seq(
+  "uk.gov.hmrc"             %% "bootstrap-play-25"        % "4.6.0",
+  "uk.gov.hmrc" %% "play-reactivemongo" % "6.2.0"
```

Comment:
Could you align this in the same way as the rest of the dependency lines? Might as well be consistent.